packages/runtime/datastore-definitions/src/channel.ts

```diff
@@ -16,6 +16,21 @@ import type { IFluidDataStoreRuntime } from "./dataStoreRuntime.js";
 import type { IChannelAttributes } from "./storage.js";
 
 /**
+ * An object which can be connected to a
+ * {@link https://fluidframework.com/docs/concepts/architecture#fluid-service|Fluid service} via a {@link IChannelServices} instance.
+ * @remarks
+ * This interfaces exposes functionality the service requires to create and maintain summaries of the channel.
+ * This summary support allows for loading a channel without having to reapply all ops that have been applied during its lifetime.
+ * @privateRemarks
+ * Since this is an interface between services (which we only expect to be implemented in this repository) and SharedObjects, which we also only expect to be implemented in this repository,
```

Comment:
Nit (to be consistent with the earlier qualifier, which is in parens)
```suggestion
 * Since this is an interface between services (which we only expect to be implemented in this repository) and SharedObjects (which we also only expect to be implemented in this repository),
```